modules/addons/src/gltf/gltf-animator.js

```diff
@@ -64,6 +64,38 @@ function applyTranslationRotationScale(gltfNode, node) {
   }
 }
 
+function linearInterpolate(path, t, a, b) {
+  // TODO: implement SLERP when path is rotation
+  return a.map((v, index) => v * (1 - t) + b[index] * t);
```

Comment:
Would recommend not creating a new array here. This function is likely to be called **a lot** in animation system.

Reply:
Changed to code to not create any new arrays.